### PR TITLE
i18n -> QR Code Reader

### PR DESCRIPTION
**Pull request by sallar**

### CODE REVIEW

**Comment by sallar** [comment]
@perry-mitchell I moved this PR here so it would go to my scanner branch first and they all go to master together later.

**Comment by perry-mitchell** [comment]
@sallar Is this still WIP?

**Comment by sallar** [comment]
@perry-mitchell no, its final.

File: source/shared/sheets.js [hunk 1]
```diff
@@ -60,7 +61,9 @@ function removeTextFromArray(arr, text) {
 export function showArchiveContentsAddItemSheet(currentGroupID) {
     const isRoot = currentGroupID == "0";
     const buttons = [...ARCHIVE_CONTENTS_ADD_ITEM_SHEET_BUTTONS];
-    const title = isRoot ? "Manage Vault" : "Edit Group";
+
+    const title = isRoot ? i18n.t("manage-vault") : i18n.t("edit-group");
+
```
Comment (perry-mitchell) [inline source/shared/sheets.js]: Guess we don't need these extra spaces?

File: source/shared/sheets.js [hunk 2]
```diff
@@ -165,16 +168,17 @@ export function showEntryPropertyValueTypeSheet() {
 export function showTouchIDToggleSheet() {
     const state = getState();
     const currentSourceID = getSelectedSourceID(state);
-    const itemEnableTouchID = "Enable Biometric Unlock";
-    const itemDisableTouchID = "Disable Biometric Unlock";
-    const itemCancel = "Cancel";
+    const itemEnableTouchID = i18n.t("biometric-unlock.enable");
+    const itemDisableTouchID = i18n.t("biometric-unlock.disable");
+    const itemCancel = i18n.t("cancel");
+
```
Comment (perry-mitchell) [inline source/shared/sheets.js]: Extra space..

**Pull request merged**